app/core/models/ocelot/stanzas/CalculationStanza.scala

```diff
@@ -63,21 +65,43 @@ sealed trait Operation {
     labels.value(ref).getOrElse("")
   }
 
-  def unsupportedOperation(operationName: String)(arg1: Any, arg2: Any): Option[String] = {
+  def valueAsOption(arg: String, labels: Labels): Option[String] = labelReference(arg).fold[Option[String]](Some(arg)){ref =>
+    labels.value(ref)
+  }
+
+  def valueAsListOption(arg: String, labels: Labels): Option[List[String]] = {
```

Comment:
valueAsListOption accessing a list label and returning as a list, wouldn't listValue be clearer as valueAsList suggests this is a conversion fn. listValue would also match value (in comment above)